packages/ia-components/sandbox/icons/customsvg/ia-logo-white-icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default props => (
```

Comment:
please add a JSDOC comment above this function to tell us what this is rendering

Reply:
okay done